reports/final/report.tex

```diff
@@ -244,6 +244,12 @@ \subsection{Software} \label{sec:soft-design}% Tim
 % Initial design: Tauri, two threaded one buffer guarded with mutex
 % Stuff from blueprint with more detail (and from ECE night)
 
+Several iterations were required to complete TeachEE's desktop application. Initially, the prototype was built using the Tauri GUI framework. Tauri was selected since it uses an instance of the Chrome Web Browser to render the user interface, allowing developers to take advantage of the rich library and tooling ecosystem for frontend web development. Despite these benefits, two major factors led us to replace Tauri: 1) displaying a rapidly changing waveform using HTML and Javascript at the required \SI{60}{\hertz} frame-rate proved extremely difficult; 2) data transfer between Rust and Javascript resulted in increased code-complexity.
```

Comment:
Suggest line breaking at 80 characters like the rest of the doc? If you are on vscode, autorewrap is a good plugin that will break text at 80 characters using an alt+q shortcut